src/index.html

```diff
@@ -7,8 +7,57 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <title>Moyo header</title>
+    <header class="header" >
+      <a href="index.html" class="logo">
+        <img
+          src="images/logo.png"
+          alt="moyo_1"
+          class="logo__img">
+      </a>
```

Comment:
still not fixed
<img width="1103" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/9e1a6e4b-01c4-46eb-93b4-b2e96285a28e">